locations/storefinders/wp_store_locator.py

```diff
@@ -79,19 +80,28 @@ def parse(self, response, **kwargs):
             item = DictParser.parse(location)
             item["street_address"] = clean_address([location.get("address"), location.get("address2")])
             item["name"] = location["store"]
-            item["opening_hours"] = self.parse_opening_hours(location)
+            # If we have preconfigured the exact days to use, start there
+            if self.days is not None:
+                item["opening_hours"] = self.parse_opening_hours(location, self.days)
+            else:
+                # Otherwise, iterate over the possibilities until we get a first match
```

Comment:
```suggestion
                # Otherwise, iterate over the possibilities until we get a first match
                logging.warning(f"Attempting to detect opening hours - specify self.days = DAYS_EN or the appropriate language code to suppress this warning")
```
Possible middle ground approach - auto detect but nag the developer?

Reply:
Sure, that works.